tracecat/expressions/functions.py

```diff
@@ -373,6 +558,27 @@ def now() -> datetime:
     return datetime.now()
 
 
+def utcnow() -> datetime:
+    """Return the current timezone-aware datetime."""
+    return datetime.now(UTC)
+
+
+def today() -> date:
+    """Return the current date."""
+    return date.today()
+
+
+def change_timezone(x: datetime, timezone: str) -> datetime:
```

Comment:
nit: I'd prefer `to_timezone` for consistency with other functions

or something like `set_timezone` and `unset_timezone`, more contrastive